addons/website_slides_survey/views/website_slides_templates_lesson_fullscreen.xml

```diff
@@ -2,9 +2,12 @@
 <odoo><data>
 
     <template id="slide_fullscreen_sidebar_category" inherit_id="website_slides.slide_fullscreen_sidebar_category">
-        <xpath expr="//li[hasclass('o_wslides_fs_sidebar_list_item')]" position="attributes">
-             <attribute name="t-att-data-certification-done">channel_progress[slide.id].get('survey_quizz_passed')</attribute>
-             <attribute name="t-att-data-certification-id">slide.survey_id.id</attribute>
+        <xpath expr="//li[@t-att-data-id='slide.id']" position="attributes">
+            <attribute name="t-att-data-certification-done">certifications_infos[slide.id]['certification_done'] if certifications_infos and slide.id in certifications_infos else None</attribute>
```

Comment:
Should make a `<t-if="certifications_infos and slide.id in certifications_infos">` for easier reading and then a `<t t-set name="slide_certification_info" value="certifications_infos[slide.id]">` for even better reading.